pyrfr/pyrfr/regression.i

```diff
@@ -29,17 +29,19 @@ typedef rfr::nodes::k_ary_node_full<2, rfr::splits::binary_split_one_feature_rss
 typedef rfr::nodes::k_ary_mondrian_node_full<2, num_t, response_t, index_t, rng_t> binary_mondrian_node_t;
 
 typedef rfr::trees::k_ary_random_tree<2, binary_full_node_rss_t, num_t, response_t, index_t, rng_t> binary_full_tree_rss_t;
+typedef rfr::trees::k_ary_random_tree<2, binary_full_node_rss_t, num_t, response_t, index_t, rng_t> binary_full_tree_rss_t_;
```

Comment:
This seems identical to the one above?

Reply:
Ignore, didn't notice the trailing underscore...seems odd that it's required for a typedef but I won't complain. Merging :)